MVP/README.md

```diff
@@ -0,0 +1,76 @@
+# Platform-CI MVP
+
+## What's MVP?
+
+Minimal Viable Product (MVP) is a minimized set of Jenkins job definition (and supporting files) on Platform CI:
+
+  - for End-to-End workflow changes (batch update) via **Template / Default YAML**
+  - for create/update/scale tests (by individuals) via **Test / Job YAML**
+
+## How to create a CI job with MVP?
+
+1. ####Get MVP files
+
+  >$ git clone https://github.com/RHQE/platform-ci.git
+
+  >$ cd platform-ci/MVP
+
+1. ####Tweak [sample_job.yaml](/MVP/sample_job.yaml/) with your own test parameters
+
+    *Job YAMLs store parameters which are specific for your testing jobs. It's recommended to maintain them in your team git repo.*
+
```

Comment:
Not sure if understand "post-MVP" correctly here, but "how to scale" is essential to MVP anyway:
- For user to scale with more jobs after their first MVP job:
  - Plan B may be easier to start with (no local JJB required) but not quite scalable with multiple job YAMLs, user will either put all job definitions in one YAML file, or run it several times with one for each YAML.
  - With Plan A, scaling from 1 to N jobs will only need user to create similar job YAMLs in the same folder. And creating/updating multiple jobs has no difference as for single job (One run for all YAMLs).
- For core team to scale MVP from v1 (brew build support) to v2 (rtt_accept compose support added), we'll enable another Template "defaults-compose"
  - if user prefer to separate build job YAMLs compose job YAMLs, they will be just working with this template similarly in a new folder
  - if user prefer to keep all build/compose job YAMLs in one place (e.g. one YAML per component, including all tiers), multiple Templates will also be able to serve without conflict.

Just some rough thoughts, and I think it worth a doc/pad for further discussion.